File: 브론즈 4/2530_인공지능 시계.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main() {
	int hour, min, sec;
	int cook_time;

	// 현재 시간, 요리 시간 입력받기
	scanf("%d %d %d", &hour, &min, &sec);
	scanf("%d", &cook_time);

	sec += cook_time;

	while (sec >= 60) {
		min++;
		sec -= 60;

		// min이 허용 범위를 초과하는 경우
		if (min >= 60)	{
			min -= 60;
			hour++;
		}

		// hour이 허용 범위를 초과하는 경우
		if (hour >= 24) {
			hour = 0;
		}
	}

	// 출력
	printf("%d %d %d", hour, min, sec);

	return 0;
}

File: 브론즈 4 (5문제)/2530 - 인공지능 시계.py
hour, minute, sec = map(int, input().split())
cook_time = int(input())

sec += cook_time

# 60초 미만으로 올바르게 시간 설정이 될 때까지 반복14
while sec >= 60:
    sec -= 60
    minute += 1

    # minute이 허용 범위를 벗어날 경우
    if minute >= 60:
        minute = 0
        hour += 1

    # hour이 허용 범위를 벗어날 경우
    if hour >= 24:
        hour = 0

print(hour, minute, sec)
